scanpipe/pipes/js.py

```diff
@@ -89,11 +92,18 @@ def source_content_sha1_list(map_file):
 
 def load_json_from_file(location):
     """Return the deserialized json content from ``location``."""
-    with open(location) as f:
-        try:
+    try:
+        with open(location) as f:
             return json.load(f)
-        except json.JSONDecodeError:
-            return
+    except UnicodeDecodeError as e:
```

Comment:
This still creates an exception and this breaks the pipeline run, we need to catch and create a ProjectMessage https://github.com/aboutcode-org/scancode.io/blob/main/scanpipe/models.py#L1399 object to keep these in the `/messages/` tab for that project, instead of raising an exception. Check similar invocations elsewhere for how to do this.

Reply:
Please run a d2d pipeline using the inputs at https://github.com/aboutcode-org/scancode.io/issues/1526#issue-2779688267, this should create an error message and the pipeline should complete successfully.